src/components/Layout.js

```diff
@@ -4,7 +4,6 @@ import { StaticQuery, graphql } from 'gatsby';
 import Meta from './Meta';
 import Nav from './Nav';
 import Footer from './Footer';
-import GithubCorner from './GithubCorner';
```

Comment:
I kind of liked the GH corner, we need to add a link back to it.
Maybe even a hire a player or something.